cli/packages/prisma-client-lib/package.json

```diff
@@ -6,8 +6,10 @@
   "files": [
     "dist"
   ],
+  "repository": "prisma/prisma",
```

Comment:
consider using the recently released [monorepo declaration](https://github.com/npm/rfcs/blob/latest/implemented/0010-monorepo-subdirectory-declaration.md), so it can point to each package inside this repo (: